WISE-Community/WISE-Client
# Support Table row selection to filter connected Graphs

## Changes
- Adds an option to Table advanced authoring to enable or disable row selection using checkboxes.
- Saves student row selections when enable.
- If Table is connected to a Graph component, Graph will only display data from selected rows.

Closes #740.

## Test
- Make sure that enabling row selection shows checkboxes to select rows in Table component.
- Make sure student row selections are saved.
- Make sure student row selections can be seen (highlighted) in show work and grading view.
- Make sure connected Graph data is filtered to only show selected rows if any rows are selected.
